doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -556,6 +560,9 @@ Bluetooth libraries and services
   * Updated the :c:func:`bt_hogp_rep_read` function to forward the GATT read error code through the registered user callback.
     This ensures that API user is aware of the error.
 
+* :ref:`bt_fast_pair_readme` library:
+  * Added support in build system for devices that do not support the :ref:`partition_manager`.
```

Comment:
```suggestion
  * Added support in build system for devices that do not support the :ref:`partition_manager`.
    The :ref:`zephyr:nrf54h20dk_nrf54h20` board target is the only example of such a device.
```

Reply:
I'd like to add this entry to NCS 2.9.0 release, because the support is already introduced in this release. @peknis, what is the process for that?

Reply:
Remove the changelog entry from this PR and add it as a comment to https://github.com/nrfconnect/sdk-nrf/pull/19240, if this is going to 2.9.0 (not 2.9.0-nRF54H20).

Reply:
Done here: https://github.com/nrfconnect/sdk-nrf/pull/19240#pullrequestreview-2489034546.